tests/test_metadata_cache.py

```diff
@@ -128,6 +129,36 @@ class TestDiskCache(CacheBaseTest, bg_test_utils.TestCaseWithFakeAccessor):
 
     CACHE_CLASS = bg_metadata_cache.DiskCache
 
+    def _get_metric_timestamp(self):
+        """Get the metric timestamp directly from DiskCache."""
+        with self.metadata_cache._DiskCache__env.begin(
+            self.metadata_cache._DiskCache__metric_to_metadata_db, write=False
+        ) as txn:
+            payload = txn.get(_TEST_METRIC.name)
+        return payload.rsplit(self.metadata_cache._METRIC_SEPARATOR, 1)[1]
```

Comment:
Please add a function in the DiskCache that does the split and use it from the tests :)